### PR TITLE
Migrate gallery to new button universe

### LINKED ISSUE #207: Update Gallery demos to use the new button universe
Now that https://github.com/flutter/flutter/pull/59702 has landed, we should update the material buttons demo to utilize them

### LINKED ISSUE #208: Update to use the new button universe
Update any buttons in studies and demos to use new button universe

### PR DESCRIPTION
**Pull request by rami-a**
This change moves all gallery demos and studies to use new button universe

Closes #207
Closes #208

### CODE REVIEW

File: lib/studies/rally/login.dart [hunk 1]
```diff
@@ -394,11 +393,15 @@ class _FilledButton extends StatelessWidget {
...
-    return FlatButton(
-      color: RallyColors.buttonColor,
-      padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 24),
-      shape: RoundedRectangleBorder(
-        borderRadius: BorderRadius.circular(12),
+    return ElevatedButton(
+      style: ElevatedButton.styleFrom(
+        primary: RallyColors.buttonColor,
+        onPrimary: Colors.black,
+        elevation: 0,
```
Comment (clocksmith) [inline lib/studies/rally/login.dart:400]: Is there some other combination in the new universe that achieves the same result as an "ElevatedButton with 0 elevation" ?

**Review by clocksmith — APPROVED** [review]

**Review by perclasson — APPROVED** [review]
Do you mind adding a screenshot of Rally and Shrine in the PR - to make sure it looks ok?

I noticed earlier that the padding of the buttons were slightly off after some change in `flutter/flutter`.

File: lib/studies/rally/login.dart (hunk 1 above)
Comment (rami-a) [reply lib/studies/rally/login.dart:400]: Yes, using a text button with a background color will achieve this too, and avoids adding elevation on press/hover, so I'll update it

**Comment by rami-a** [comment]
Added, they look right to me I think

Screen Shot 2020-10-13 at 10 47 32 AM
Screen Shot 2020-10-13 at 10 48 19 AM
Screen Shot 2020-10-13 at 10 48 24 AM

**Comment by perclasson** [comment]
The height of the Rally button looks slightly wrong (it's outside of the scope of this PR probably). It should look something like this:

Screen Shot 2020-10-13 at 16 56 14

**Comment by rami-a** [comment]
> The height of the Rally button looks slightly wrong (it's outside of the scope of this PR probably). It should look something like this:
> 
> Screen Shot 2020-10-13 at 16 56 14

I think it is out of scope, checking rally on master and it looks the same to me:

Screen Shot 2020-10-13 at 11 04 49 AM

**Pull request merged**